.github/workflows/build.yml

```diff
@@ -60,9 +60,15 @@ jobs:
           dpkg-query -W --showformat='${Package}\n' 'postgresql-*' | xargs sudo dpkg -P postgresql
 
           wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
-          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main 13" >> /etc/apt/sources.list.d/pgdg.list'
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main 14" >> /etc/apt/sources.list.d/pgdg.list'
           sudo apt-get update -qq
           sudo apt-get install -qq postgresql-${{ matrix.pg_major }}
+          export PGDATA=/etc/postgresql/${{ matrix.pg_major }}/main
+          # Starting with PG14 the default password encryption mode is scram.
+          # Changing it to md5 for compatability.
+          if [ ${{ matrix.pg_major }} -ge 14 ]; then
+            sudo sed -i 's/#password_encryption = scram-sha-256/password_encryption = md5/' $PGDATA/postgresql.conf
```

Comment:
Shouldn't we just have a pg_hba.conf which sets the encryption explicitly? This way we don't have to worry about before and after PG14...

Also, this would allow us to have one user with MD5 and another with SCRAM. We could use environment variables to communicate to the tests to test these different authentication methods (I don't think they're tested at the moment).

Reply:
It doesn't work that way - the user's password is encrypted the moment it's created and it's not re-created on `password_encryption`'s change. And you can't connect with md5 to a user with a scram encrypted password (and vice versa).

Reply:
I know. I meant we'd have two different users: test_md5 and test_scram (or whatever), and use environment variables to tell each test to run (the tests would be skipped when the environment variables are absent, so that on developer machines they don't cause failures).

Reply:
Yeah, we can do this. Although I'm not going to do this today (maybe in the next few days).